src/sdk/namespace_fs.js

```diff
@@ -1471,6 +1477,7 @@ class NamespaceFS {
         dbg.log1('Namespace_fs._move_to_dest_version:', new_ver_tmp_path, latest_ver_path, upload_file);
         let gpfs_options;
         const is_gpfs = native_fs_utils._is_gpfs(fs_context);
+        const is_dir = this._is_directory_content(latest_ver_path, key);
```

Comment:
same here - the naming is confusing, is_dir_content is more accurate, there can be a dir that is not a content dir